common/traffic_light_utils/include/traffic_light_utils/traffic_light_utils.hpp

```diff
@@ -24,7 +24,7 @@
 #include <tf2/LinearMath/Matrix3x3.h>
 #include <tf2/LinearMath/Transform.h>
 
-namespace perception_utils
+namespace traffic_light_utils
 {
 
 namespace traffic_light
```

Comment:
@miursh This namespace seems redundant if the first namespace is changed from `perception_utils` to `traffic_light_utils`